snuba/query/processors/custom_function.py

```diff
@@ -70,12 +70,12 @@ class CustomFunction(QueryProcessor):
 
     def __init__(
         self,
-        dataset_schema: ColumnSet,
+        entity: Entity,
```

Comment:
Here as well you should not need this as `process_query` can pass the entity data source to `SignatureValidator` and `SignatureValidator` can extract the column set from there.